logistics.ipynb

```diff
@@ -3,8 +3,6 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    new_code = 2*5
</pre><p>is this good</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/magnus-3009/test2/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>